common/persistence/sql/sqlplugin/mysql/admin.go

```diff
@@ -70,7 +70,7 @@ func (mdb *db) CreateSchemaVersionTables() error {
 // ReadSchemaVersion returns the current schema version for the keyspace
 func (mdb *db) ReadSchemaVersion(database string) (string, error) {
 	var version string
-	err := mdb.driver.Get(sqlplugin.DbAllShards, &version, readSchemaVersionQuery, database)
+	err := mdb.driver.Get(sqlplugin.DbDefaultShard, &version, readSchemaVersionQuery, database)
```

Comment:
why change this?

Reply:
Because there is difficulty implementing `All shards` in current driver interface. I let it read from a single shard(database) for now and was planning to improve it as a follow-up: https://github.com/uber/cadence/issues/4509

Reply:
If you like I can also try to implement it here -- there are different ways of doing it -- the simpliest way is to send the queries to all shards and make sure all of them return the same value(deep equal). 

Other ideas: 1) allow setting shardID when sending query, or return a set of reset instead of a single one. -- both would require a refactor on the code path. 

I feel like it's better do it in a separate PR. What do you think ?

Reply:
Sure, let's do it in a separate PR.